drivers/n25q128/include/n25q128_internal.h

```diff
@@ -0,0 +1,237 @@
+
+/*
+ * @brief   Quad Input/Output Fast Read
+ * NOTE: Only available in Quad mode, which is not supported yet.
+ */
+#define N25Q128_OPCODE_QIOFR        (0xEB)
+
+/*
+ * @brief   Read OTP (Read of OTP area)
+ */
+#define N25Q128_OPCODE_ROTP         (0x4B)
+
+/*
+ * @brief   Write Enable
+ */
+#define N25Q128_OPCODE_WREN         (0x06)
+
+/*
+ * @brief   Write Disable
+ */
+#define N25Q128_OPCODE_WRDI         (0x04)
+
+/*
+ * @brief   Page Program
+ */
+#define N25Q128_OPCODE_PP           (0x02)
+
+/*
+ * @brief   Dual Input Fast Program
+ * NOTE: Only available in Dual mode, which is not supported yet.
+ */
+#define N25Q128_OPCODE_DIFP         (0xA2)
+
+/*
+ * @brief   Dual Input Extended Fast Program
+ * NOTE: Only available in Dual mode, which is not supported yet.
+ */
+#define N25Q128_OPCODE_DIEFP        (0xD2)
+
+/*
+ * @brief   Quad Input Fast Program
+ * NOTE: Only available in Quad mode, which is not supported yet.
+ */
+#define N25Q128_OPCODE_QIFP         (0x32)
+
+/*
+ * @brief   Quad Input Extended Fast Program
+ * NOTE: Only available in Quad mode, which is not supported yet.
+ */
+#define N25Q128_OPCODE_QIEFP        (0x12)
+
+/*
+ * @brief   Program OTP (Program of OTP area)
+ */
+#define N25Q128_OPCODE_POTP         (0x42)
+
+/*
+ * @brief   SubSector Erase
+ */
+#define N25Q128_OPCODE_SSE          (0x20)
+
+/*
+ * @brief   Sector Erase
+ */
+#define N25Q128_OPCODE_SE           (0xD8)
+
+/*
+ * @brief   Bulk Erase
+ */
+#define N25Q128_OPCODE_BE           (0xC7)
+
+/*
+ * @brief   Program/Erase Resume
+ */
+#define N25Q128_OPCODE_PER          (0x7A)
+
+/*
+ * @brief   Program/Erase Suspend
+ */
+#define N25Q128_OPCODE_PES          (0x75)
+
+/*
+ * @brief   Read Status Register
+ */
+#define N25Q128_OPCODE_RDSR         (0x05)
+
+/*
+ * @brief   Write Status Register
+ */
+#define N25Q128_OPCODE_WRSR         (0x01)
+
+/*
+ * @brief   Read Lock Register
+ */
+#define N25Q128_OPCODE_RDLR        (0xE8)
+
+/*
+ * @brief   Write to Lock Register
+ */
+#define N25Q128_OPCODE_WRLR         (0xE5)
+
+/*
+ * @brief   Read Flag Status Register
+ */
+#define N25Q128_OPCODE_RFSR         (0x70)
+
+/*
+ * @brief   Clear Flag Status Register
+ */
+#define N25Q128_OPCODE_CLFSR        (0x50)
+
+/*
+ * @brief   Read NV Configuration Register 1
+ */
+#define N25Q128_OPCODE_RDNVCR       (0xB5)
+
+/*
+ * @brief   Write NV Configuration Register
+ */
+#define N25Q128_OPCODE_WRNVCR       (0xB1)
+
+/*
+ * @brief   Read Volatile Configuration Register
+ */
+#define N25Q128_OPCODE_RDVCR        (0x85)
+
+/*
+ * @brief   Write Volatile Configuration Register
+ */
+#define N25Q128_OPCODE_WRVCR        (0x81)
+
+/*
+ * @brief   Read Volatile Enhanced Configuration Register
+ */
+#define N25Q128_OPCODE_RDVECR       (0x65)
+
+/*
+ * @brief   Write Volatile Enhanced Configuration Register
+ */
+#define N25Q128_OPCODE_WRVECR       (0x61)
+
+/** }@ Extended SPI OPCODE */
+
+/*
+ * @brief   Legacy SPI Status Register of the N25Q128 device.
+ */
```

Comment:
I guess this needs doxygen grouping. Try to build the documentation (`make doc` from RIOT root directory) and you will see all doxygen errors and warnings.